recipe/meta.yaml

```diff
@@ -49,4 +49,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - jp
+    - kujaku11
+    - kkappler
```

Comment:
@kkappler are you ok being a maintainer here?

(asking as this a new maintainer that was not previously on the maintainer list)

Reply:
@jakirkham I'm glad to be a maintainer - thanks for asking